### DISCUSSION TITLE
Partial Prompt Inclusion and Initial Audio Omission in Transcript

### DISCUSSION THREAD

**Discussion by CMoranoA** [openai/whisper#1486] [Q&A]
I'm experiencing a specific issue. I have a large audio file, which I've cut into chunks of about 15 minutes each, taking silences into account to avoid cutting off a speaker mid-sentence. However, sometimes at the beginning of the next audio transcription, part of the prompt appears in the transcript while the first minute and a half of the audio do not.

I tried reducing the audio chunk durations to 10 minutes, and also experimented with reducing and modifying the prompt. But the problem persists.

Has anyone else encountered this problem? Any solutions would be greatly appreciated!
Thank you!

**Comment by nickaldred** [comment]
Hey, did you find a solution for this?

**Comment by beResonant** [comment]
I experience this too, almost for certain if I am using the 'tiny' model, and there is excessive silence at the beginning or end of the file (or segment, for those chopping their audio down before processing).

If I trim them to have zero silence, use at least the base model, and manually transcribe the first 40-50 characters of the prompt to match the actual content exactly, it doesn't seem to happen. And doing the brief manual transcription seems to help with hallucinations generally. I'll follow that up with a semi-colon, and then whatever other guide text I want after that in "natural language"
